program_transformers/src/token_metadata/v1_asset.rs

```diff
@@ -446,7 +435,7 @@ async fn find_model_with_retry<T: ConnectionTrait + TransactionTrait, K: EntityT
 fn record_metric(metric_name: &str, success: bool, retries: u32) {
     let retry_count = &retries.to_string();
     let success = if success { "true" } else { "false" };
-    metric! {
-        statsd_count!(metric_name, 1, "success" => success, "retry_count" => retry_count);
+    if cadence_macros::is_global_default_set() {
+        cadence_macros::statsd_count!(metric_name, 1, "success" => success, "retry_count" => retry_count);
```

Comment:
The metric! macro auto does the check, could we still use that to remain consistent?

```
macro_rules! metric {
    {$($block:stmt;)*} => {
        if is_global_default_set() {
            $(
                $block
            )*
        }
    };
}
```

Reply:
define macro locally? otherwise we would depend from `nft_ingester` here :confused:

Reply:
I see now.  Nevermind it's good as is!